.github/actions/helm-publish-chart/action.yml

```diff
@@ -86,10 +89,16 @@ runs:
       env:
         BASE_URL: ${{ steps.build-base-url-option.outputs.result }}
         SUB_REPO: ${{ steps.build-destination-path.outputs.result }}
+        RELEASE_PACKAGE_NAME: ${{ inputs.chart-package }}
+        RELEASE_PACKAGE_PATH: ${{ inputs.chart-package-path }}
       run: |
           mkdir $RANDOM_PATH
-          RELEASE_PACKAGE_NAME=${{ inputs.chart-package }}
-          cp "$RELEASE_PACKAGE_NAME" $RANDOM_PATH
+          if [ -n "$RELEASE_PACKAGE_PATH" ]
+          then
+            cp "$RELEASE_PACKAGE_PATH" $RANDOM_PATH
+          else
+            cp "$RELEASE_PACKAGE_NAME" $RANDOM_PATH
+          fi
```

Comment:
don't know if is going to break too many things, but you can drop `chart-package` input and just have the `chart-package-path` input and when you need the name of the file only you can extract it with `basename ${{ inputs.chart-package-path }}`

Reply:
that could work too because in the end `file.ext` is still a valid path, but make sure to make it clear in the doc of the input that is accepting a path as well